model/Core/Vocabularies/RelationshipType.md

```diff
@@ -81,5 +81,5 @@ between a Package and a File, between two Packages, or between one SPDXDocument
 - testTool: Every `to` Element is a test tool for the `from` Element
 - testedOn: (AI, Dataset) The `from` Element has been tested on the `to` Element
 - trainedOn: (AI, Dataset) The `from` Element has been trained by the `to` Element(s)
-- underInvestigationFor: (Security/VEX) The impact of a vulnerability is being investigated
+- underInvestigationFor: (Security/VEX) The `from` Vulnerability element impact is being investigated for each `to` Element
```

Comment:
s/Vulnerability element/Vulnerability/

For the "to" Element - Should this be "to" Artifact?    We need to be consistent here I think?

Reply:
@puerco - can you have a VEX underInvestigationFor relationship type for an SBOM? or is just for the software product/artifact? We chose `Element` here if a vulnerability could be under investigation for an SBOM/Element Collection